science/volk/Portfile

```diff
@@ -24,9 +24,7 @@ long_description {*}${long_description} ${subport} \
 license             GPL-3
 homepage            https://libvolk.org/
 
-github.setup gnuradio volk 2.5.0 v
-# Change github.tarball_from to 'releases' or 'archive' next update
-github.tarball_from tarball
```

Comment:
please read the comment and set it to "releases" (preferred, if upstream provides them) or otherwise to "archive".